### PR TITLE
Google Drive Backup & more fixes

### PR DESCRIPTION
**Pull request by brunobar79**
This makes the Google drive backup feature working on Android.

Also:
- includes the new onboarding flow (WelcomeScreen + new empty wallet state)
- Fixes the import sheet layout
- Fixes build issues on release mode 
- Fixes the fonts and missing SFSymbols across the app

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "version": "1.2.35-1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
+    "android": "adb uninstall me.rainbow || true && react-native run-android",
```
Comment (osdnk) [inline package.json:6]: why this? It should override old build

File: patches/react-native-keychain+6.1.1.patch [hunk 2]
```diff
@@ -96,26 +96,4000 @@ index 98e45fe..937e179 100644
  RCT_EXPORT_METHOD(resetInternetCredentialsForServer:(NSString *)server
                    resolver:(RCTPromiseResolveBlock)resolve
                    rejecter:(RCTPromiseRejectBlock)reject)
+diff --git a/node_modules/react-native-keychain/android/build/.transforms/220bd5e945f10aebb2b06c3e13aae822.bin b/node_modules/react-native-keychain/android/build/.transforms/220bd5e945f10aebb2b06c3e13aae822.bin
```
Comment (osdnk) [inline patches/react-native-keychain+6.1.1.patch:99]: how about this file? Why it >3000 lines?

File: src/utils/platform.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { Platform } from 'react-native';
+
+export const CLOUD_PLATFORM = Platform.OS === 'ios' ? 'iCloud' : 'Google Drive';
```
Comment (osdnk) [inline src/utils/platform.js:3]: I think it should be more "cloudPlatform". It's not constant i thik. But i don't have a strong opinion

**Review by osdnk — APPROVED** [review]

File: patches/react-native-keychain+6.1.1.patch (hunk 2 above)
Comment (brunobar79) [reply patches/react-native-keychain+6.1.1.patch:99]: I'll re-apply the patch without including the build folder.

File: package.json (hunk 1 above)
Comment (brunobar79) [reply package.json:6]: It doesn't when you switch from release mode to develop or viceversa and it's very annoying cause you have to manually delete app and re-run the entire command again

**Pull request merged**